examples/opf/tools/dataGenerationTool.py

```diff
@@ -22,8 +22,8 @@
 
```

Comment:
@scottpurdy Did you forget fix this file name! how about `data_generator.py` instead of `DataGenerationTool.py`?